packages/types/src/interfaces/extrinsic/signedExtensions/types.ts

```diff
@@ -3,8 +3,8 @@ import { InterfaceTypes } from '@polkadot/types/types';
 export type ExtTypes = Record<string, keyof InterfaceTypes>;
 
 export type ExtInfo = {
-  extra: ExtTypes;
-  types: ExtTypes;
+  extrinsic: ExtTypes;
```

Comment:
Updated the types to use same as upstream... so that we can store the signedExtension as it is...